specification/network/resource-manager/Microsoft.Network/2017-06-01/virtualNetworkGateway.json

```diff
@@ -1514,39 +1564,54 @@
           }
         },
         "ingressBytesTransferred": {
-          "readOnly": true,		
+          "readOnly": true,
           "type": "integer",
           "format": "int64",
           "description": "The Ingress Bytes Transferred in this connection"
         },
         "egressBytesTransferred": {
-          "readOnly": true,		
+          "readOnly": true,
           "type": "integer",
           "format": "int64",
           "description": "The Egress Bytes Transferred in this connection"
-        },		
+        },
         "lastConnectionEstablishedUtcTime": {
-          "readOnly": true,		
+          "readOnly": true,
           "type": "string",
           "description": "The time at which connection was established in Utc format."
         }
       },
       "description": "VirtualNetworkGatewayConnection properties"
     },
+    "GatewayReference": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID."
```

Comment:
nit: Resource usually refers to a top level ARM resource, please provide a more detailed description.